dune_api_scripts/queries/retention-on-date.sql

```diff
@@ -21,7 +21,7 @@ retention_data as (
     from cow_traders
     left outer join dex.trades
         on trader = trader_a
-        and project != 'Gnosis Protocol'
+        and project != 'CoW Protocol'
```

Comment:
Does this need to be an `not in ('CoW Protocol', 'Gnosis Protocol')` (since the latter appears to be in the `dex.trades` table still).

Reply:
![image](https://user-images.githubusercontent.com/4210206/183451637-b4241641-0857-40d6-affd-024cc70ced33.png)

Reply:
That is gnosis protocol v1